docs/pages/index.html

```diff
@@ -20,11 +20,11 @@
   snippet: |
     @model gdemo(x, y) = begin
```

Comment:
We could use the opportunity and change the example to the syntax that is more common in the implementation andd documentation nowadays:
```suggestion
    @model function gdemo(x, y)
```